lab-dan/routes/dog-routes.js

```diff
@@ -0,0 +1,62 @@
+'use strict'
+
+const responseHandler = require('../lib/responseHandler')
+
+module.exports = (router, storage) => {
```

Comment:
I like that you passed the storage object in here, just on principle of working with dep injection and getting a feel for how it works. 

I would argue that the context of Routers means that you have a module that can and should be reused by passing it into model-specific routes using dep injecction. That being said, the storage module and model constructor are specific to the model-routes, so a hard-coupling (requiring/importing) is more appropriate as opposed to dependency injection.